mkosi/sandbox.py

```diff
@@ -239,43 +239,43 @@ def sandbox_cmd(
     return cmdline
 
 
-def apivfs_cmd(root: Path) -> list[PathString]:
+def apivfs_cmd() -> list[PathString]:
     return [
         "bwrap",
         "--dev-bind", "/", "/",
-        "--tmpfs", root / "run",
-        "--tmpfs", root / "tmp",
-        "--bind", "/var/tmp", root / "var/tmp",
-        "--proc", root / "proc",
-        "--dev", root / "dev",
+        "--tmpfs", "/buildroot/run",
+        "--tmpfs", "/buildroot/tmp",
+        "--bind", "/var/tmp", "/buildroot/var/tmp",
+        "--proc", "/buildroot/proc",
+        "--dev", "/buildroot/dev",
         # Make sure /etc/machine-id is not overwritten by any package manager post install scripts.
-        "--ro-bind-try", root / "etc/machine-id", root / "etc/machine-id",
+        "--ro-bind-try", "/buildroot/etc/machine-id", "/buildroot/etc/machine-id",
         # Nudge gpg to create its sockets in /run by making sure /run/user/0 exists.
-        "--dir", root / "run/user/0",
-        *flatten(mount.options() for mount in finalize_passwd_mounts(root)),
+        "--dir", "/buildroot/run/user/0",
+        *flatten(mount.options() for mount in finalize_passwd_mounts("/buildroot")),
         "sh", "-c",
-        f"chmod 1777 {root / 'tmp'} {root / 'var/tmp'} {root / 'dev/shm'} && "
-        f"chmod 755 {root / 'run'} && "
+        "chmod 1777 /buildroot/tmp /buildroot/var/tmp /buildroot/dev/shm && "
+        "chmod 755 /buildroot/run && "
         # Make sure anything running in the root directory thinks it's in a container. $container can't always be
         # accessed so we write /run/host/container-manager as well which is always accessible.
-        f"mkdir -m 755 {root}/run/host && echo mkosi >{root}/run/host/container-manager && "
+        "mkdir -m 755 /buildroot/run/host && echo mkosi >/buildroot/run/host/container-manager && "
         "exec $0 \"$@\"",
```

Comment:
The warning is wrong, since the implicit concatenation is intended here, but maybe something like this
```python
        " && ".join(
            [
                "chmod 1777 /buildroot/tmp /buildroot/var/tmp /buildroot/dev/shm",
                "chmod 755 /buildroot/run",
                "mkdir -m 755 /buildroot/run/host && echo mkosi >/buildroot/run/host/container-manager",
                "exec $0 \"$@\"",
            ],
        )
```
or 
```python
        (
            "chmod 1777 /buildroot/tmp /buildroot/var/tmp /buildroot/dev/shm && " +
            "chmod 755 /buildroot/run && " +
            "mkdir -m 755 /buildroot/run/host && echo mkosi >/buildroot/run/host/container-manager && " +
            "exec $0 \"$@\""
        ),
```
(with or without the `+` operators) would make the inline nature of this script more apparent.

Also same below for the other warning.